v1.0/deploy-cockroachdb-to-production.md

```diff
@@ -0,0 +1,424 @@
+---
+title: How to Deploy CockroachDB to Production
+summary: Get an overview of how best to deploy CockroachDB as a consistent, scalable database for your application.
+toc: false
+---
+
+Though CockroachDB itself is easy to use, deploying a database to production requires a lot of preparation. This page attempts to cover common things you need to understand before deploying CockroachDB––or any service––to a production environment.
+
+{{site.data.alerts.callout_info}}For help training your team on CockroachDB's deployments or best practices, <a href="https://www.cockroachlabs.com/pricing/sales/">contact the Cockroach Labs team</a>.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Deployment Architecture
+
+To deploy CockroachDB to production, you need the following infrastructure.
+
+#### Nodes
+
+3+ machines that can communicate with one another.
+
+- Use at least three nodes to ensure that a majority of replicas (2/3) remains available if a node fails. Configurations with odd numbers of replicas are more robust than those with even numbers. Clusters of three and four nodes can each tolerate one node failure and still reach a majority (2/3 and 3/4 respectively), so the fourth replica doesn't add any extra fault-tolerance. To survive two simultaneous failures, you must have five replicas.
+
+- Run each node on a separate machine. Since CockroachDB replicates across nodes, running more than one node per machine increases the risk of data loss if a machine fails. Likewise, if a machine has multiple disks or SSDs, run one node with multiple `--store` flags and not one node per disk. For more details about stores, see [Start a Node](start-a-node.html).
+
+- Ideally, these machines should be located in you would want these machines located in separate locations to maximize survivability.
```

Comment:
uh. I think you mean `these machines should be located in separate locations ...`